src/webots/nodes/WbTriangleMeshGeometry.cpp

```diff
@@ -0,0 +1,697 @@
+// Copyright 1996-2020 Cyberbotics Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "WbTriangleMeshGeometry.hpp"
+
+#include "WbBoundingSphere.hpp"
+#include "WbCoordinate.hpp"
+#include "WbField.hpp"
+#include "WbFieldChecker.hpp"
+#include "WbMFInt.hpp"
+#include "WbMatter.hpp"
+#include "WbNodeUtilities.hpp"
+#include "WbNormal.hpp"
+#include "WbOdeGeomData.hpp"
+#include "WbRay.hpp"
+#include "WbResizeManipulator.hpp"
+#include "WbSFBool.hpp"
+#include "WbSFDouble.hpp"
+#include "WbSFNode.hpp"
+#include "WbSimulationState.hpp"
+#include "WbTextureCoordinate.hpp"
+#include "WbTransform.hpp"
+#include "WbTriangleMesh.hpp"
+#include "WbWrenMeshBuffers.hpp"
+#include "WbWrenRenderingContext.hpp"
```

Comment:
It seems this is a copy and paste from `WbIndexedFaceSet`, but here some of these includes are useless (for example WbCoordinate, WbMFInt, etc.)

Also in `WbIndexedFaceSet.cpp` some of the include are no longer needed because the code has been moved here.